inspektor/lint.py

```diff
@@ -64,6 +64,7 @@ def get_opts(self):
             if self.args.enable:
                 pylint_args.append('--enable=%s' % self.enabled_errors)
             pylint_args.append('--reports=no')
+            pylint_args.append('--errors-only')
```

Comment:
NACK:

```
       --errors-only, -E
              In  error mode, checkers without error messages are disabled and
              for others, only  the  ERROR  messages  are  displayed,  and  no
              reports are done by default
```

When trying with a simple file:

```
[medic@localhost inspektor ]$ cat a.py
import os

print "A"
[medic@localhost inspektor ]$ inspekt lint a.py 
Pylint disabled: W,R,C,E1002,E1101,E1103,E1120,F0401,I0011
Pylint enabled : W0611
Syntax check PASS
```
When I disable `--errors-only` it works well:

```
[medic@localhost inspektor ]$ inspekt lint a.py 
Pylint disabled: W,R,C,E1002,E1101,E1103,E1120,F0401,I0011
Pylint enabled : W0611
************* Module a
W0611:  1,0: : Unused import os
Pylint check fail: a.py
Syntax check FAIL
```